browser/src/control/Control.PartsPreview.js

```diff
@@ -617,29 +617,10 @@ L.Control.PartsPreview = L.Control.extend({
 
 	_onScroll: function (e) {
 		setTimeout(L.bind(function (e) {
```

Comment:
e becomes unused after this, so there's a warning that the build catches now I think

Reply:
Thanks, fixed. I do not why my local build show as error